tests/integration/defs/accuracy/test_disaggregated_serving.py

```
@@ -315,6 +334,69 @@ def test_eagle3(self, overlap_scheduler):
             task = GSM8K(self.MODEL_NAME)
             task.evaluate(llm)
 
+    def run_parallel_test(self, ctx_pp: int, ctx_tp: int, gen_pp: int,
+                          gen_tp: int, test_set: LlmapiAccuracyTestHarness):
+        if ctx_tp * ctx_pp + gen_tp * gen_pp > get_device_count():
+            pytest.skip(
```

```suggestion
            pytest.fail(
```

---

What's the reason for failing instead of skipping?